.travis.yml

```diff
@@ -83,8 +91,8 @@ script:
   - if [[ $TRAVIS_PULL_REQUEST ]]; then
       export NOSEATTR="!nonpublic";
     fi
-  - nosetests indra -v --with-doctest --with-doctest-ignore-unicode
-        --with-coverage --cover-inclusive --cover-package=indra -a $NOSEATTR
+  - nosetests -v -s --process-restartworker indra/tests/test_tees.py -a $NOSEATTR
+  - python travis_continual_output.py nosetests indra -v  -s -a $NOSEATTR -e '.*tees.*'
```

Comment:
Replace this with simple call to nosetests with all the options e.g --with-coverage above.